corehq/apps/sms/static/sms/js/sms_usage_report.js

```diff
@@ -0,0 +1,15 @@
+hqDefine("sms/js/sms_usage_report",[
+    'jquery',
+    'knockout',
```

Comment:
Doesn't look like you need knockout here.

Reply:
Yep. I removed it from parameters but forgot to remove it from here.